debezium-server/debezium-server-http/src/test/java/io/debezium/server/http/HTTPTestResourceLifecycleManager.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.server.http;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.testcontainers.containers.GenericContainer;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class HTTPTestResourceLifecycleManager implements QuarkusTestResourceLifecycleManager {
```

Comment:
```suggestion
public class HttpTestResourceLifecycleManager implements QuarkusTestResourceLifecycleManager {
```

Here and potentially elsewhere (it's the project convention for abbreviations in identifiers).

Reply:
Good to know. Wasn't too sure how best to handle the HTTP case.